.github/CONTRIBUTING.md

```diff
@@ -101,6 +101,52 @@ A good commit may look like this:
 
     [#42] Discover the meaning of life.
 
+### Code Reviews
+
+* Always be constructive when providing feedback (e.g. be explicit on
+  what could improved and how)
+* Use [GitHub suggestion blocks][github-suggest] for fixing typos and
+  small mistakes
+* Positive feedback is as important as negative feedback
+* Only the Pull Request author(s) should push changes to the branch
+  under review
```

Comment:
Do we want to say anything about a rebase policy?

Reply:
GitHub seems to handle most of the cases by itself. We can add something, yes. Maybe as a follow-up?